### PR TITLE
Update Hotfixes for BfA

### PR DESCRIPTION
**Pull request by funjoker**
Split from other BfA Update

### CODE REVIEW

File: WowPacketParser/Hotfix/IHotfixSerializer.cs [hunk 1]
```diff
@@ -61,13 +63,14 @@ static IHotfixSerializer()
...
-                serializationEmitter.LoadConstant(isString ? @"""{0}"", " : "{0}, ");
+                serializationEmitter.LoadConstant(isString ? @"'{0}', " : "{0}, ");
                 serializationEmitter.LoadArgument(0); // instance
                 serializationEmitter.CallVirtual(propInfo.GetGetMethod());
                 if (isString)
                 {
-                    serializationEmitter.LoadConstant(@"\""");
-                    serializationEmitter.LoadConstant(@"""");
+                    serializationEmitter.Call(stringEscape);
+                    serializationEmitter.LoadConstant(Environment.NewLine);
```
Comment (Warpten) [inline WowPacketParser/Hotfix/IHotfixSerializer.cs]: Wait what. Why would you have newlines put out (also newlines in double quoted strings are fine in SQL)

Comment (funjoker) [reply WowPacketParser/Hotfix/IHotfixSerializer.cs]: It's to unify all SQL output.
We used to use single quote: '
And make newlines \n
Also we escaping strings by using MySqlHelper

**Comment by Warpten** [comment]
IL is lgtm

**Comment by funjoker** [comment]
hotfix_data and hotfixes should actually be together, shouldn't they?

**Comment by funjoker** [comment]
Included in #436

**Pull request closed without merging**